src/api/payment-requests/payment-requests.md

```diff
@@ -354,6 +354,7 @@ Payment Activities are created when a Payment Request has been **created**, **pa
           classification: {
             type: 'GS1',
             code: '10001874',
+            name: 'CROCKERY',
```

Comment:
I chose this as the example as in the GS1 examples:
1. Titles are capitalised 
2. Titles describe the category of a product